plugins/ruby-foreman-google/debian/control

```diff
@@ -0,0 +1,24 @@
+Source: ruby-foreman-google
+Section: ruby
+Priority: optional
+Maintainer: Leos Stejskal <lstejska@redhat.com>
+Build-Depends: debhelper (>= 11~),
+               gem2deb,
+               rake,
+               ruby-google-cloud-compute (>= 0.2)
+Standards-Version: 4.3.0
+Homepage: https://github.com/theforeman/foreman_google
+Testsuite: autopkgtest-pkg-ruby
+XS-Ruby-Versions: all
+Rules-Requires-Root: no
+
+Package: ruby-foreman-google
+Architecture: all
+XB-Ruby-Versions: ${ruby:Versions}
+Depends: ruby | ruby-interpreter,
+         ruby-google-cloud-compute (>= 0.2),
+         foreman (>= 3.4.0~rc1), 
```

Comment:
3.4 is not a thing yet, so this wouldn't work. what do you need from future 3.4? 3.3 would at least match nightly